P5/Source/Specs/unicodeName.xml

```diff
@@ -28,10 +31,20 @@ $Id$
     <textNode/>
     <!--s:assert> some schematron rules here</s:assert-->
   </content>
+  <constraintSpec scheme="schematron" validUntil="2021-07-09"
```

Comment:
Delete the entire `<constraintSpec>`. The deprecation Schematron check will (or at least should) be automatically generated at build time because of the `@validUntil` on the `<elementSpec>`.